metadata.rb

```diff
@@ -5,7 +5,7 @@
 license 'MIT'
 description 'Set of chef recipes for OpsWorks based Ruby projects'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version '0.2.1'
+version '0.3.0'
```

Comment:
Bumped this to 0.3.0 because the change to removing sqlite as a default would be considered a breaking change. It seems too minor a change to wait for a full version revision.